README.md

```diff
@@ -1 +1,185 @@
+- [📗 Table of Contents](#-table-of-contents)
+- [🎥 RSL Media ](#-rsl-media-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [How to access the database?](#how-to-access-the-database)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 🎥 RSL Media <a name="about-project"></a>
+
+>
+> **The RSL Media** is a Microverce group project. It's an online entertainment streaming source for TV shows, movies. All data is preserved thanks to the external **tvmaze** API service.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> We used HTML, CSS, Javascript ES6, Webpack and Jest Unit testing.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JAVASCRIPT</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+> Some key feature of this project
+Features
+- **User can add like to their favourite show**
+- **User can add comments**
+- **User can reserve a show**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> ## Project demo picture
+- [Live Demo Link](https://raihan2bd.github.io/js-capstone/)
+>
+> <div>
+>
+  <img src="src/assets/img/comment.png" alt="comment-popup-window" width="500" height="300" />
+  <img src="src/assets/img/reservation.png" alt="reservation-popup-window" width="500"  height="300" />
+   <img src="src/assets/img/home.png" alt="home-page" width="500" height="300/>
+  <div>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+>Please follow the instructions to clone the repo
+To get a local copy, follow these simple example steps.
+Clone this repository or download the Zip folder:
+
+### Prerequisites
+
+>In order to check the linters errors make sure you have installed the [nodejs](https://nodejs.org)
+
+
+### Setup
+
+>Clone this repository to your desired folder: cd [folder] to navigate and run the below comand to clone the project
+
+```sh
+git clone https://github.com/raihan2bd/js-capstone.git
+```
+
+
+### Install
+
+Install this project with:
+
+download npm the packages
+```sh
+npm install --save-dev
+```
+
+### Usage
+
+ > To see the project in your local browser first of all, go to the dist folder then open the index.html file.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+
+
+👤 **Abu Raihan**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)

```

Comment:
-[ ]**[Optional] these links are not valid 😨** I think it would be better to provide existing links to your social media accounts for the best user experience and this will make your README file more professional 👍 👍